server/api/events.spec.js

```diff
@@ -0,0 +1,108 @@
+const {expect} = require('chai')
+const request = require('supertest')
+const db = require('../db')
+const app = require('../index')
+const Event = db.model('Event')
+
+describe.only('Event routes', () => {
+  beforeEach(() => {
+    return db.sync({force: true})
+  })
+  describe('Event GET routes', () => {
+    beforeEach(async () => {
+      const event1 = await Event.create({
+        description: 'An Event from 1995',
+        name: 'My Event',
+        date: new Date('December 17, 1995 03:24:00'),
+        status: 'done'
+      })
+      const event2 = await Event.create({
+        description: 'An Event from 2001',
+        name: '2001 Event',
+        date: new Date('December 17, 2001 03:24:00'),
+        status: 'done'
+      })
+    })
+    it('GETS all events /api/events', async () => {
+      const res = await request(app)
+        .get('/api/events')
+        .expect(200)
+
+      expect(res.body).to.be.an('array')
+      expect(res.body.length).to.be.equal(2)
+    })
+    it('GET for single /api/events/:eventID', async () => {
+      const res = await request(app)
+        .get('/api/events/1')
+        .expect(200)
+
+      expect(res.body).to.be.an('object')
+      expect(res.body.name).to.be.equal('My Event')
+      expect(res.body.status).to.equal('done')
+    })
+    it('GET for single, search by Name /api/events/:eventName', async () => {
+      const res = await request(app)
+        .get('/api/events/name/2001 Event')
+        .expect(200)
+
+      expect(res.body).to.be.an('array')
+      expect(res.body[0].name).to.be.equal('2001 Event')
+      expect(res.body[0].status).to.equal('done')
+    })
+  })
+  describe('PUT Routes', () => {
+    beforeEach(async () => {
+      await Event.create({
+        name: 'Event To Edit',
+        status: 'pending',
+        description: 'Ipsum Lorae'
+      })
+    })
+    it('changes name', async () => {
+      return request(app)
+        .put('/api/events/1')
+        .send({name: 'EventUpdated', status: 'pending'})
+        .expect(res => {
+          expect(res.body.name).to.be.equal('EventUpdated')
+          expect(res.body.status).to.be.equal('pending')
+          //description should be set to null
+          expect(res.body.description).to.be.equal(null)
```

Comment:
Kinda surprised these pass without returning:true set on the endpoint db queries. Am I missing something?